### ISSUE TITLE
No detection when converting detectron2 model that use Lazy Configs

### ISSUE THREAD

**Issue by ha-ves** [NVIDIA/TensorRT#4428]
## Description

I successfuly exported and converted Detectron2 Mask RCNN R50-FPN ONNX into TensorRT and built an engine but it doesn't detect or very very rarely detects anything, even if it does it's so far below, barely above base NMS (~0.254) score.

Used 1344x1344 input image tensor
I tried exporting ONNX with opset 11 12 15 16 20 they all same result.

ONNXRuntime detects just fine and similar mAP but slower inference time by ~20%.

## Environment

**TensorRT Version**:
TensorRT-10.9.0.34

**NVIDIA GPU**:
T1000 Mobile

**NVIDIA Driver Version**:
572.60

**CUDA Version**:
12.6

**CUDNN Version**:
9.6.0

Operating System:
Windows 11 24H2 26100.3775

Python Version (if applicable):
3.11

PyTorch Version (if applicable):
2.6.0+cu12

## Relevant Files

<details>
<summary>changed part of create_onnx.py</summary>

```
        # Set up Detectron 2 model configuration.
        # lazy config
        with open(config_file, 'rb') as f:
            self.det2_cfg = cloudpickle.load(f)

        self.det2_model = instantiate(self.det2_cfg.model)
        self.det2_model.to(self.det2_cfg.train.device)
        checkpointer = DetectionCheckpointer(self.det2_model)
        checkpointer.load(weights)
        self.det2_model.eval()

        self.fpn_out_channels = self.det2_cfg.model.backbone.out_channels
        self.num_classes = self.det2_cfg.model.roi_heads.num_classes

        self.first_NMS_max_proposals = self.det2_model.proposal_generator.post_nms_topk[False] # is train = False
        self.first_NMS_iou_threshold = self.det2_cfg.model.proposal_generator.nms_thresh
        self.first_NMS_score_threshold = 0.01 # 1%

        self.first_ROIAlign_pooled_size = self.det2_cfg.model.roi_heads.box_pooler.output_size
        self.first_ROIAlign_sampling_ratio = self.det2_cfg.model.roi_heads.box_pooler.sampling_ratio
        self.first_ROIAlign_type = self.det2_cfg.model.roi_heads.box_pooler.pooler_type
        
        self.second_NMS_max_proposals = self.det2_model.roi_heads.box_predictor.test_topk_per_image if max_det is None else max_det
        self.second_NMS_iou_threshold = self.det2_model.roi_heads.box_predictor.test_nms_thresh
        self.second_NMS_score_threshold = self.det2_cfg.model.roi_heads.box_predictor.test_score_thresh

        self.second_ROIAlign_pooled_size = self.det2_cfg.model.roi_heads.mask_pooler.output_size
        self.second_ROIAlign_sampling_ratio = self.det2_cfg.model.roi_heads.mask_pooler.sampling_ratio
        self.second_ROIAlign_type = self.det2_cfg.model.roi_heads.mask_pooler.pooler_type

        self.mask_out_res = 28
```
</details>

config file is [new_baselines/mask_rcnn_R_50_FPN_100ep_LSJ.py](https://github.com/facebookresearch/detectron2/blob/main/configs/new_baselines/mask_rcnn_R_50_FPN_100ep_LSJ.py)

**Model link**:
resulting converted onnx

![Image](https://github.com/user-attachments/assets/0ff7f923-e1e6-4e93-a22a-cc25dfe6e4dc)

## Steps To Reproduce
detectron2 export -> ONNX -> convert for TensorRT -> build engine

**Comment by yuanyao-nv** [comment]
Could you also try running the ONNX model with onnxruntime to verify the model is exported correctly?
